client/src/components/forms/MailingList/index.tsx

```diff
@@ -85,6 +86,19 @@ function MailingListForm({ classes }: WithStyles<typeof styles>) {
     }
   };
 
+  const submit = async () => {
+    try {
+      await mailingListSubmit({
+        email,
+        interestedEvents: events.map((event) => event.title),
+        otherFeedback: feedback,
+      });
+      goRight();
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
```

Comment:
What a beauty😍

Reply:
If for any reason it fails the user can open up their console to see why 😏 
#UX